File: math/divisors.py
import math

def get_divisors(n):
    divisors = set()
    for i in range(1, int(math.sqrt(n)) + 1):
        if n % i == 0:
            divisors.add(i)
            divisors.add(n // i)
    return sorted(list(divisors))

# Example usage
number = 36
print(f"Divisors of {number}: {get_divisors(number)}")

File: math/divisors.go
package main

import (
	"fmt"
	"math"
	"sort"
)

func getDivisors(n int) []int {
	var divisors []int
	sqrtN := int(math.Sqrt(float64(n)))

	for i := 1; i <= sqrtN; i++ {
		if n%i == 0 {
			divisors = append(divisors, i)
			// Avoid adding the square root twice
			if i*i != n {
				divisors = append(divisors, n/i)
			}
		}
	}
	sort.Ints(divisors)
	return divisors
}

func main() {
	number := 36
	fmt.Printf("Divisors of %d: %v\n", number, getDivisors(number))
}